tasks/ModelTask.cpp

```diff
@@ -222,20 +222,37 @@ void ModelTask::updateLinks(base::Time const& time)
     for(ExportedLinks::const_iterator it = exported_links.begin(); it != exported_links.end(); ++it)
     {
         math::Pose source2world = math::Pose::Zero;
+        math::Vector3 sourceInWorld_linear_vel  = math::Vector3::Zero;
+        math::Vector3 sourceInWorld_angular_vel = math::Vector3::Zero;
         if (it->second.source_link_ptr)
+        {
             source2world = it->second.source_link_ptr->GetWorldPose();
+            sourceInWorld_linear_vel  = it->second.source_link_ptr->GetWorldLinearVel();
+            sourceInWorld_angular_vel = it->second.source_link_ptr->GetWorldAngularVel();
+        }
+
         math::Pose target2world = math::Pose::Zero;
         if (it->second.target_link_ptr)
             target2world = it->second.target_link_ptr->GetWorldPose();
+
         math::Pose source2target( math::Pose(source2world - target2world) );
+        math::Vector3 sourceInTarget_angular_vel(target2world.rot.RotateVectorReverse(sourceInWorld_angular_vel));
```

Comment:
Yeah. I'll update to stick to that until we resolve https://github.com/rock-core/base-types/pull/100